spot-ml/SPARKCONF.md

```diff
@@ -39,16 +39,18 @@ file exist a section for _Spark_ properties, below is the explanation for each o
             SPK_DRIVER_MAX_RESULTS=''   ---> Maximumn driver results in MB or GB i.e. 8g
             SPK_EXEC_CORES=''           ---> Cores per executor i.e. 4
             SPK_DRIVER_MEM_OVERHEAD=''  ---> Driver memory overhead in MB i.e. 3047. Note that there is no "m" at the end.
-            SPAK_EXEC_MEM_OVERHEAD=''   ---> Executor memory overhead in MB i.e. 3047. Note that there is no "m" at the end.
+            SPK_EXEC_MEM_OVERHEAD=''    ---> Executor memory overhead in MB i.e. 3047. Note that there is no "m" at the end.
+            SPK_AUTO_BRDCST_JOIN_THR='10485760' ---> Spark's spark.sql.autoBroadcastJoinThreshold. Default is 10MB, increase this value to make Spark broadcast tables larger than 10 MB and speed up joins.
+            SCALING_OPTION='64'         ---> Indicates Spot whether to use Double precision probabilities or Float.  
```

Comment:
Yeah, there is one new section for that actually.  It's called 
##### Spark autoBroadcastJoinThreshold in spot-ml